### PR TITLE
DMP-357 Default TimeZone: UTC (Database: TYPE TIMESTAMP WITH TIME ZONE)

### PR DESCRIPTION
**Pull request by richardbull**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DMP-357

### Change description ###

Default TimeZone: UTC
https://tools.hmcts.net/confluence/display/DMP/5.+Common+format+for+timestamps
* Database: TYPE TIMESTAMP WITH TIME ZONE / spring.jpa.properties.hibernate.jdbc.time_zone=UTC
* Service / JPA Entity: ISO_OFFSET_DATE_TIME / OffsetDateTime

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/darts/audio/controller/AudioRequestController.java [hunk 1]
```diff
@@ -35,7 +35,7 @@ public class AudioRequestController implements AudioRequestsApi {
         operationId = "addAudioRequest",
         summary = "Users can request audio for specific cases and date/time periods",
         description = "Adds a user audio request for processing",
-        tags = { "Audio Requests" },
+        tags = { "AudioRequests" },
```
Comment (jackmaloney) [inline src/main/java/uk/gov/hmcts/darts/audio/controller/AudioRequestController.java:38]: How come the space has been removed here?

File: src/main/resources/db/migration/V1_4__update_timestamp_with_time_zone.sql [hunk 2]
```diff
@@ -0,0 +1,19 @@
+ALTER TABLE notification ALTER created_date_time TYPE TIMESTAMP WITH TIME ZONE;
```
Comment (jackmaloney) [inline src/main/resources/db/migration/V1_4__update_timestamp_with_time_zone.sql:1]: Do we want this script? Or can we just blow the whole database away and start fresh?

File: src/main/java/uk/gov/hmcts/darts/audio/controller/AudioRequestController.java (hunk 1 above)
Comment (richardbull) [reply src/main/java/uk/gov/hmcts/darts/audio/controller/AudioRequestController.java:38]: The OAS currently uses tags to group the operations. This was the gotcha I asked @cmagowan-hmcts about and he mentioned in the dev channel - https://hmcts-reform.slack.com/archives/C04T0CVJ28N/p1685530019602349

File: src/integrationTest/java/uk/gov/hmcts/darts/audio/AudioRequestServiceTest.java [hunk 3]
```diff
@@ -16,44 +16,107 @@
...
+
         List<AudioRequest> resultList = audioRequestRepository.findByRequestId(requestId);
         AudioRequest requestResult = resultList.get(0);
         assertTrue(requestResult.getRequestId() > 0);
-        assertEquals("OPEN", requestResult.getStatus());
-        assertEquals(caseId, requestResult.getCaseId());
+        assertEquals(OPEN_STATUS, requestResult.getStatus());
+        assertEquals(requestDetails.getCaseId(), requestResult.getCaseId());
+        assertEquals(requestDetails.getStartTime(), requestResult.getStartTime());
+        assertEquals(requestDetails.getEndTime(), requestResult.getEndTime());
```
[the remark this answers is not in the document]
Comment (richardbull) [reply src/integrationTest/java/uk/gov/hmcts/darts/audio/AudioRequestServiceTest.java:117]: I can change it, if you are finding it difficult to read. I thought it was easier to just check the response matches the request fields myself.

[the remark this answers is not in the document]
Comment (jackmaloney) [reply src/integrationTest/java/uk/gov/hmcts/darts/audio/AudioRequestServiceTest.java:117]: I guess it's fine.

**Review by jackmaloney — APPROVED** [review]

**Pull request merged**